dusty/payload.py

```diff
@@ -19,10 +20,11 @@ def run(self):
         self.fn(*self.args, **self.kwargs)
 
     def serialize(self):
-        doc = {'fn': self.fn, 'client_version': self.client_version, 'args': self.args, 'kwargs': sorted(self.kwargs.items())}
```

Comment:
Deterministic behavior is generally nice. Don't think we're relying on it for anything though.